### DISCUSSION TITLE
After install, No Boot Device Found on Dell Lattitude

### DISCUSSION THREAD

**Discussion by KerryatCLC** [elementary/triage#275] [Q&A]
I installed OS 7 Horus on a Dell Lattitude 3590 to SSD.  On reboot, NO BOOT DEVICE FOUND. Any ideas?

**Comment by lenemter** [comment]
How did you partitioned your drive? Or did you do a full disk install?

**Reply by KerryatCLC** [reply]
Full wipe and install

**Comment by davidmhewitt** [comment]
I've saw this once on one of my old Dell laptops after installing Ubuntu. Sometimes the Dell BIOS/EFI doesn't pick up the new boot config for some reason.

If you enter the EFI/BIOS setup and go into the boot config, do you have an option to add elementary OS to the boot order? Or the option to create a new boot option and select the drive/EFI file for elementary OS? Feel free to post photos and I can talk you through it if it's not obvious.

**Reply by KerryatCLC** [reply]
I tried to add a boot option and I got a Warning box saying File System Not Found! [OK]

**Comment by RogerWeihrauch** [comment]
Hi all
Donno if it is solved already.

I experienced two ways to make the boot code accessible in Efi/Bios Boot option:
1) add the efi boot image manually to the secure boot DB
2) add id using the mocking option with generating secure boot keys and adding them in Bios after reboot

for1) this can only be done if secure boot is Disabled in UEFI BIOS
for2) this is a bit a more complex workout:
may these links could help/give a more transparency:
https://unix.stackexchange.com/questions/565615/efi-boot-bootx64-efi-vs-efi-ubuntu-grubx64-efi-vs-boot-grub-x86-64-efi-gru
https://askubuntu.com/questions/1085550/cant-install-ubuntu-18-10-on-xps-15-efi-boot-mmx64-efi-not-found
Regards,
Roger
